R/Task.R

```diff
@@ -951,6 +953,10 @@ Task = R6Class("Task",
     #'   Not more than a single column can be associated with this role.
     #' * `"stratum"`: Stratification variables. Multiple discrete columns may have this role.
     #' * `"weight"`: Observation weights. Not more than one numeric column may have this role.
+    #' * `"offset"`: Offset values specifying fixed adjustments for model training.
+    #'   These values can be used to provide baseline predictions from an existing model for updating another model.
+    #'   Some learners require an offset for each target class in a multiclass setting.
+    #'   In this case, the offset columns must be named `"offset_{target_class_name}"`.
     #'
```

Comment:
Simple models just shift the prediction. And add link to learners.

Reply:
link? https://mlr3learners.mlr-org.com/reference/mlr_learners_classif.xgboost.html or [mlr3learners::LearnerClassifXgboost]?